src/components/lists/Lists.vue

```diff
@@ -0,0 +1,211 @@
+          <va-item-section avatar>
+            <va-avatar>
+              <img :src="customer.picture" :alt="customer.name">
+            </va-avatar>
+          </va-item-section>
+
+          <va-item-section>
+            <va-item-label>
+              {{ customer.name }}
+            </va-item-label>
+
+            <va-item-label caption>
+              {{ customer.address }}
+            </va-item-label>
+          </va-item-section>
+
+          <va-item-section side>
+            <va-icon name="fa fa-eye" color="gray" />
+          </va-item-section>
+        </va-item>
+
+        <va-list-separator v-if="i < customers.length - 1" :key="'separator' + customer.id" />
+      </template>
+    </va-list>
+
+    <va-list fit style="maxWidth: 400px" class="mb-2">
+      <va-list-label>
+        {{ $t('lists.recentMessages') }}
+      </va-list-label>
+
+      <template v-for="(customer, i) in customers">
+        <va-item :key="'item' + customer.id" clickable>
+          <va-item-section side>
+            <va-icon v-if="customer.starred" name="fa fa-star" color="warning" />
+          </va-item-section>
+
+          <va-item-section avatar>
+            <va-avatar>
+              <img :src="customer.picture" :alt="customer.name">
+            </va-avatar>
+          </va-item-section>
+
+          <va-item-section>
+            <va-item-label>
+              {{ customer.name }}
+            </va-item-label>
+          </va-item-section>
+
+          <va-item-section side>
+            <va-icon
+              :name="getGenderIcon(customer.gender)"
+              :color="getGenderColor(customer.gender)"
+            />
+          </va-item-section>
+        </va-item>
+
+        <va-list-separator v-if="i < customers.length - 1" :key="'separator' + customer.id" />
+      </template>
+
+      <va-list-separator fit spaced />
+
+      <va-list-label color="gray">
+        {{ $t('lists.archieved') }}
+      </va-list-label>
+
+      <template v-for="(customer, i) in archived">
+        <va-item :key="'item' + customer.id">
+          <va-item-section side>
+            <va-icon v-if="customer.starred" name="fa fa-star" color="warning" />
+          </va-item-section>
+
+          <va-item-section avatar>
+            <va-avatar>
+              <img :src="customer.picture" :alt="customer.name">
+            </va-avatar>
+          </va-item-section>
+
+          <va-item-section>
+            <va-item-label>
+              {{ customer.name }}
+            </va-item-label>
+          </va-item-section>
+        </va-item>
+
+        <va-list-separator v-if="i < archived.length - 1" :key="'separator' + customer.id" />
+      </template>
+    </va-list>
+
+    <va-list fit style="maxWidth: 400px">
+      <va-list-label>
+        {{ $t('lists.starterKit') }}
+      </va-list-label>
+
+      <va-item clickable>
+        <va-item-section>
+          <va-item-label>
+            Add profile images
```

Comment:
Shouldn't all these (this one and below) strings be localized?

Reply:
I wouldn't worry too much about secondary labels like that one

Reply:
We may also receive some community support for things like this